cmd/minishift/cmd/config/config.go

```diff
@@ -23,6 +23,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/spf13/viper"
+
```

Comment:
This line and line no 29 is unnecessary IMO.

Reply:
? Are you sure, Please check

https://github.com/anjannath/minishift/blob/ef25d03f25c6656d6b335f0809abf26a3ddfe55b/cmd/minishift/cmd/config/config.go#L154

https://github.com/anjannath/minishift/blob/ef25d03f25c6656d6b335f0809abf26a3ddfe55b/cmd/minishift/cmd/config/config.go#L160

Reply:
@budhrg Those are added by the IDE i am using :). fixing those.